go-controller/pkg/node/iprulemanager/ip_rule_manager.go

```diff
@@ -28,13 +28,17 @@ type Controller struct {
 
 // NewController creates a new linux IP rule manager
 func NewController(v4, v6 bool) *Controller {
-	return &Controller{
+	nc := &Controller{
 		mu:            &sync.Mutex{},
 		rules:         make([]ipRule, 0),
 		ownPriorities: make(map[int]bool, 0),
 		v4:            v4,
 		v6:            v6,
 	}
+	if err := nc.loadHostRules(); err != nil {
```

Comment:
I dont think this is good because you are loading rules that we may not own and then rule manager will try to persist them and those rules maybe added by other components outside ovn-k.

Ideally before a controller shuts down it would delete them but that isnt going to happen because it causes disruption, so I added the "own" a priority func which allows a controller to define what it expects to see there.

Reply:
Delete is supposed to represent - stop managing a rule that was added and delete it. 
If the controller that uses this IP rules manager owns a priority because it cannot clean up IP rules when it exists, it doesnt need to call delete, it can just add what it wants to be present at a given priority.

Reply:
bump @andreaskaris ^^